src/app/_components/navbar.tsx

```diff
@@ -27,44 +27,92 @@ export function Navbar() {
 					</div>
 
 					<div className="hidden md:flex justify-end gap-6">
+						<a href="/posts" className="text-gray-700 hover:text-blue-500 transition duration-300">
+							Articles
+						</a>
+						<a
+							href="/releases"
+							className="text-gray-700 hover:text-blue-900 transition duration-300"
+						>
+							Releases
+						</a>
 						<a
 							href="mailto:support@smswithoutborders.com"
-							target="_blank"
-							rel="noopener noreferrer"
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
 						>
-							<FaEnvelope size={20} className="rounded" />
+							<FaEnvelope size={30} className="rounded bg-white p-1" />
 						</a>
-						<a href={FACEBOOK_URL} target="_blank" rel="noopener noreferrer">
-							<FaFacebook size={20} className="rounded" />
+						<a
+							href={FACEBOOK_URL}
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
+						>
+							<FaFacebook size={30} className="rounded-full bg-white p-1" />
 						</a>
-						<a href={X_URL} target="_blank" rel="noopener noreferrer">
-							<FaXTwitter size={20} className="rounded" />
+						<a href={X_URL} className="text-gray-700 hover:text-blue-500 transition duration-300">
+							<FaXTwitter size={30} className="rounded-full bg-white p-1" />
 						</a>
-						<a href={GITHUB_URL} target="_blank" rel="noopener noreferrer">
-							<FaGithub size={20} className="rounded" />
+						<a
+							href={GITHUB_URL}
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
+						>
+							<FaGithub size={30} className="rounded-full bg-white p-1" />
 						</a>
-						<a href={LINKEDIN_URL} target="_blank" rel="noopener noreferrer">
-							<FaLinkedin size={20} className="rounded" />
+						<a
+							href={LINKEDIN_URL}
+							className="text-gray-700 hover:text-blue-500 transition duration-300"
+						>
+							<FaLinkedin size={30} className="rounded-full bg-white p-1" />
```

Comment:
We should ensure that all items have a uniform style, including text size, colours, and effects.